clients/ui/frontend/src/components/DashboardDescriptionListGroup.scss

```diff
@@ -0,0 +1,4 @@
+.kubeflow-custom-description-list-term-with-action > span {
+  /* Workaround for missing functionality in PF DescriptionList, see https://github.com/patternfly/patternfly/issues/6583 */
+  width: 100%;
```

Comment:
@Griffin-Sullivan have you tried getting rid of this? it might be solved in patternfly 6.

Reply:
no let me take a look at that before we merge

Reply:
Ok yeah this CSS is needed or else the edit button will be touching the subheading.